include/fuzzuf/mutator/mutator.hpp

```diff
@@ -654,8 +687,15 @@ void Mutator<Tag>::Havoc(u32 stacking, const std::vector<AFLDictData> &extras,
 
       default:
         custom_cases(r, outbuf, len, extras, a_extras);
+        selected_case_histogram[r] += 1;
```

Comment:
In the default case, r can be greater than the size of `selected_case_histogram`, can't it?
Is it no problem?

Reply:
fixed in 621077e8c314ae8959f7057d19c82b3026068b60